workbench-core/audit/src/baseExtractor.test.ts

```diff
@@ -14,7 +13,7 @@ describe('BaseExtractor', () => {
   const mockMethod = 'GET';
   const mockUrl = '/sample';
   const sampleIp = 'sampleIP';
-  const mockUser: AuthenticatedUser = {
+  const mockUser = {
```

Comment:
why is the type being removed?

Reply:
To remove dependency on authorization, it would be a circular dependency audit -> authz -> audit